src/Algebra/Bundles.agda

```diff
@@ -892,6 +892,12 @@ record NonAssociativeRing c ℓ : Set (suc (c ⊔ ℓ)) where
   open AbelianGroup +-abelianGroup public
     using () renaming (group to +-group; invertibleMagma to +-invertibleMagma; invertibleUnitalMagma to +-invertibleUnitalMagma)
 
+  *-unitalMagma : UnitalMagma _ _
+  *-unitalMagma = record { isUnitalMagma = *-isUnitalMagma}
```

Comment:
Shouldn't this actually already be available from `AbelianGroup`? If not, then it should be added there rather than here right?

Reply:
Abelian group is for "addition" and we add here for "multiplication". (Or am I again missing something?)

Reply:
Ah nope, you're right, my mistake, sorry!